.nvmrc

```
@@ -0,0 +1 @@
+16
```

nit: just to get rid of the little ⛔ 
```suggestion
16

```